### PR TITLE
Update security.md

### PR DESCRIPTION
**Pull request by darrelmiller**
Fix security examples as the security property is an array.

### CODE REVIEW

**Comment by kevinswiber** [comment]
@darrelmiller I think this also needs to be fixed on lines 58, 142, and 188.

File: specification/security.md [hunk 1]
```diff
@@ -139,14 +139,14 @@ info:
...
-  oauth2Profiles:
+  - oauth2Profiles:
     - board:read
     - board:write
 paths:
   /board:
     get:
-      security:
-        oauth2Profiles: []
+      - security:
```
Comment (kevinswiber) [inline specification/security.md:148]: @darrelmiller This hyphen needs to be removed.

**Review by kevinswiber — CHANGES_REQUESTED** [review]
@darrelmiller Just one little hiccup. Otherwise, LGTM!

**Comment by karenetheridge** [comment]
"defaultApiKey" is not one of the valid entries for a securityScheme. The schema currently only allows for one of: apiKey, http, mutualTLS, oauth2, openIdConnect.

**Comment by kevinswiber** [comment]
> "defaultApiKey" is not one of the valid entries for a securityScheme. The schema currently only allows for one of: apiKey, http, mutualTLS, oauth2, openIdConnect.

@karenetheridge I believe this is the `name` in the Security Requirement Object, which must correspond to a name in `#/components/securitySchemes`. The example shows a Security Scheme Object in Components that's called `defaultApiKey`, so I think this might be correct.

**Comment by karenetheridge** [comment]
Aha, my confusion came from "The name used for each property MUST correspond to a security scheme declared in the [Security Schemes](https://spec.openapis.org/oas/v3.1.0#componentsSecuritySchemes) under the [Components Object](https://spec.openapis.org/oas/v3.1.0#components-object)." -- which I interpreted as the `type`, not the property name under components.

**Comment by handrews** [comment]
@karenetheridge 

> Aha, my confusion came from "The name used for each property MUST correspond to a security scheme declared in the [Security Schemes](https://spec.openapis.org/oas/v3.1.0#componentsSecuritySchemes) under the [Components Object](https://spec.openapis.org/oas/v3.1.0#components-object)." -- which I interpreted as the type, not the property name under components.

Ah, good catch.  The wording could perhaps be something like "The name used for each property MUST correspond to the name of a property under the `securitySchemes` field of the Components Object."

File: specification/security.md (hunk 1 above)
Comment (ralfhandl) [inline specification/security.md:148]:
```suggestion
      security:
```

**Review by ralfhandl — APPROVED** [review]

**Review by mikekistler — APPROVED** [review]
Looks good. 👍

**Pull request merged**